Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: build test
+
+build:
+	docker-compose up -d --build
+test:
+	@echo "Running tests..."
+	docker compose -f docker-compose.yml run app pytest -s -vv
```

Comment:
docker compose (bez - ) jest oki ale w build masz uzyte docker-compose z -. ujednolicic to mozna 
mozna tez dodac cel lint ktory odpala black i ruff skoro masz je w rqueirements 
lint:
 black . 
ruff .